src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,31 @@
 import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { LoaderEnum } from '../types/loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
 
 export const PeoplePage = () => {
+
+  const [error, setError] = useState('');
+  const [loader, setLoader] = useState<LoaderEnum>(
+    LoaderEnum.initial
+  );
+
+  const [people, setPeople] = useState<Person[]>([]);  useEffect(() => {
+    setLoader(LoaderEnum.loading);
+
+
+    getPeople().then((data) => {setPeople(data)})
+      .catch(error => {
+        setError(error);
+      })
```

Comment:
fix eslint errors.
I remember you have some problems with run eslint locally and you use webstorm IDE, but check settings in webstorm which can help to detect errors during development.
<img width="967" alt="Screenshot 2025-05-08 at 14 20 29" src="https://github.com/user-attachments/assets/f8a6c09e-f131-47f9-8080-80c377106707" />
<img width="967" alt="Screenshot 2025-05-08 at 14 20 51" src="https://github.com/user-attachments/assets/40f7edac-066a-4c4f-8631-7cc3168aa1d8" />

All this errors are relevant and need to be fixed.
![Screenshot 2025-05-08 at 14 19 24](https://github.com/user-attachments/assets/56ee0ab1-eaa4-4253-8e72-b60224d87508)

https://github.com/mate-academy/react_people-table-advanced/actions/runs/14903450125/job/41860381559?pr=1361